<launch>

    <arg name="input_topic" default="/rslidar_points_unfiltered"/>
    <arg name="output_topic" default="/rslidar_points"/>
    <arg name="x_min" default="-0.42"/> <!--0.39+0.03-->
    <arg name="x_max" default="0.42"/> <!--0.39+0.03-->
    <arg name="y_min" default="-0.31"/> <!--0.28+0.03-->
    <arg name="y_max" default="0.31"/> <!--0.28+0.03-->
    <arg name="z_min" default="-0.1"/>
    <arg name="z_max" default="1.0"/>

    <node
        pkg="lidar_filter"
        type="lidar_filter.py"
        name="lidar_filter"
        output="screen">
        <param name="input_topic" value="$(arg input_topic)"/>
        <param name="output_topic" value="$(arg output_topic)"/>
        <param name="x_min" value="$(arg x_min)"/>
        <param name="x_max" value="$(arg x_max)"/>
        <param name="y_min" value="$(arg y_min)"/>
        <param name="y_max" value="$(arg y_max)"/>
        <param name="z_min" value="$(arg z_min)"/>
        <param name="z_max" value="$(arg z_max)"/>
    </node>

</launch>